config/routes.rb

```diff
@@ -12,9 +12,10 @@
 
     namespace :account do
       resources :users, except: %i[index destroy]
-      resources :utility_providers, only: %i[index new update] do
+      resources :utility_providers, only: %i[index new show update] do
         get 'search', on: :collection
         put 'disassociate', on: :member
+        resources :payments, only: %i[show]
```

Comment:
If you have only one action, you can just specify that one, you dont need recourses